extensions/amp-ad-network-cloudflare-impl/0.1/test/test-amp-ad-network-cloudflare-impl.js

```diff
@@ -124,7 +125,8 @@ describe('amp-ad-network-cloudflare-impl', () => {
     });
 
     it('should handle both data parameters and templated parameters', () => {
-      el.setAttribute('src', '/ad?width=SLOT_WIDTH&height=SLOT_HEIGHT');
+      el.setAttribute('src', 'https://firebolt.cloudflaredemo.com' +
```

Comment:
Are there some tests that it correctly fails if the `src` does not start with the Cloudflare base URL?  (Or, at least, is not a fully qualified URL?)

Also, there should be tests for the 'optionally convert to a4a endpoint' logic.